benches/enrichment_tables_file.rs

```diff
@@ -0,0 +1,123 @@
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
```

Comment:
👍 thanks for adding these benches.

Would you want to add them to the `benches.yml` Github Actions workflow too?

Reply:
I think we will probably want to update them before they become "official" benchmarks. These largely prove that indexing trumps sequential scans, so it isn't really worth comparing sequential scans in future.